testsuite/tests/kuadrant/authorino/dinosaur/conftest.py

```diff
@@ -282,3 +288,57 @@ def user_with_write_role(admin_rhsso, blame):
     user = admin_rhsso.realm.create_user(blame("someuser"), blame("password"))
     user.assign_realm_role(admin_rhsso.realm.create_realm_role("admin-write"))
     return HttpxOidcClientAuth.from_user(admin_rhsso.get_token, user=user)
+
+
+@pytest.fixture(scope="module")
+def delete_jwks_key(rhsso):
+    """Deletes signing RS256 key from JWKS"""
+
+    def _delete_jwks_key():
+        jwks = rhsso.realm.admin.get_keys()["keys"]
+        assert jwks is not None
+
+        provider_id = None
+        for key in jwks:
+            if key["use"] == "SIG" and key["algorithm"] == "RS256" and key["status"] == "ACTIVE":
+                provider_id = key["providerId"]
+                break
+        assert provider_id is not None
+
+        rhsso.realm.admin.delete_component(provider_id)
+
+    return _delete_jwks_key
+
+
+@pytest.fixture(scope="module")
+def create_jwks_key(rhsso):
+    """Creates a new signing RS256 key in JWKS"""
+
+    def _create_jwks_key():
```

Comment:
how about turning `_create_jwks_key` and `_delete_jwks_key` into methods of RHSSO class instead? they seem pretty general even though I don't think it is used anywhere else in the testsuite. WDYT?

Reply:
Thanks! That sounds like a good idea at the first glance. I will look into it and try to refactor as suggested.

Reply:
Refactored and force pushed. Please re-review if you can spare some time. Thanks!